presto-main/src/main/java/com/facebook/presto/execution/StageStats.java

```diff
@@ -80,6 +81,7 @@ public StageStats()
         this.queuedDrivers = 0;
         this.runningDrivers = 0;
         this.completedDrivers = 0;
+        this.cumulativeMemoryReservation = 0.0;
```

Comment:
It's unclear to me what the difference is between "cumulative" and "total" memory. Maybe rename this to "memoryReservationTimeIntegral"?